PiwikTracker/PiwikTracker.swift

```diff
@@ -105,15 +106,20 @@ final public class PiwikTracker: NSObject {
                     self.dispatchBatch()
                 })
             }, failure: { shouldContinue in
-                self.logger.warning("Failed dispatching events with error \(shouldContinue)")
```

Comment:
Not part of this PR but what is `shouldContinue`? From the name I would say it's a `Bool`, but it is logged in the warning as error?